### ISSUE TITLE
Ghostscript crashes

### ISSUE THREAD

**Issue by jmairboeck** [haikuports/haikuports#6284]
Ghostscript crashes in two testcases of fig2dev (see #6283)
[gs-6083-debug-16-10-2021-15-44-27.txt](https://github.com/haikuports/haikuports/files/7359602/gs-6083-debug-16-10-2021-15-44-27.txt)

**Comment by Begasus** [comment]
There were no crashes on 32bit with the current ghostscript version

**Comment by jmairboeck** [comment]
Version 9.55.0 shows the same crashes (and other issues) as version 9.50.

**Comment by jmairboeck** [comment]
ghostscript8 is not available for x86_64, only for x86. Should we enable it also for other architectures (and maybe update it a bit as well)?

Is the reason why this exists the license?

**Comment by pulkomandy** [comment]
No, the reason is it is needed by old BeOS apps. There is no need to port it over to x86_64 please. Use up to date versions there.
